### PR TITLE
Several improvements to the community page

### PR DESCRIPTION
**Pull request by beorn7**
See commit descriptions for details.

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -111,9 +111,9 @@ <h1>Commercial Support</h1>
 
     <h1>Acknowledgements</h1>
     <p>
-      Prometheus was initially started by
+      Prometheus was started by
       <a href="http://www.matttproud.com">Matt T. Proud</a> and
-      <a href="http://juliusv.com">Julius Volz</a>. The majority of its
+      <a href="http://juliusv.com">Julius Volz</a>. The majority of its initial
       development has been sponsored by <a href="https://soundcloud.com">SoundCloud</a>.
```
Comment (juliusv) [inline content/community.html]: maybe also "has been" -> "was"? I don't know why, just sounds like even more "over" :)

File: content/community.html [hunk 2]
```diff
@@ -106,9 +111,9 @@ <h1>Commercial Support</h1>
...
```
Comment (juliusv) [inline content/community.html]: maybe also "has been" -> "was"? I don't know why, just sounds like even more "over" :)

**Comment by juliusv** [comment]
:+1:

File: content/community.html [hunk 3]
```diff
@@ -45,10 +45,15 @@ <h1>Community</h1>
     <h1>Contributing</h1>
     <p>
       We welcome community contributions! Please see the
-      <code>CONTRIBUTING.md</code> file in the respective Prometheus repository
-      for instructions on how to submit changes. If you are planning on making
-      more elaborate or controversial changes, please discuss them on the mailing
-      list before sending a pull request.
+      <code>CONTRIBUTING.md</code> file in the respective Prometheus
+      repository for instructions on how to submit changes. If you are
+      planning on making more elaborate or controversial changes,
```
Comment (brian-brazil) [inline content/community.html]: Should we say potentially controvrsial?

**Comment by beorn7** [comment]
I agree with both points. Will update and push. Thanks, everyone.

**Pull request merged**